config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -578,6 +578,18 @@ spec:
               image_pull_secret:  # deprecated
                 description: (Deprecated) Image pull secret for app and database containers
                 type: string
+              host_aliases:
+                description: HostAliases for app containers
+                type: array
+                items:
+                  type: object
+                  properties:
+                    ip:
+                      type: string
+                    hostnames:
+                      type: array
+                      items:
+                        type: string
               task_resource_requirements:
```

Comment:
Could you also add an entry in awx-operator.clusterserviceversion.yaml for this so that the parameter shows up in the OLM UI form?

Here is an example:
* https://github.com/ansible/awx-operator/blob/devel/config/manifests/bases/awx-operator.clusterserviceversion.yaml#L25-L28

These docs might be useful:
* https://github.com/openshift/console/blob/447e90085090543e670adfe8e9eb935312346dfa/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#20-fielddependency

Reply:
I added this [entry](https://github.com/ansible/awx-operator/pull/1232/files#diff-2a4968792da0598da409c1e5bbc604542cdf7b2b52bbbf4179dd018fcf9baaf4), but I'm not sure if it's right.